README.md

```diff
@@ -32,7 +32,7 @@ A longer example, demonstrating how to invoke the CoffeeScript compiler:
 ``` ruby
 require "execjs"
 require "open-uri"
-source = open("http://coffeescript.org/extras/coffee-script.js").read
+source = URI.open("https://coffeescript.org/browser-compiler-legacy/coffeescript.js").read
```

Comment:
While we're at it, we shouldn't suggest using open-uri in 2024.

Reply:
```ruby
require "net/http"
source = Net::HTTP.get(URI("https://coffeescript.org/browser-compiler-legacy/coffeescript.js"))
```

Reply:
@byroot updated.